From e4a3abaaef03b17ab2b7ab2a3f6f7ad251cf2133 Mon Sep 17 00:00:00 2001
From: Surflurer <22912139+Surflurer@users.noreply.github.com>
Date: Mon, 22 Jan 2024 18:42:14 +0800
Subject: [PATCH] Do not check for last world info

---
 src/worldfactory.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/worldfactory.cpp b/src/worldfactory.cpp
index 9861ce8238..a0c559ee96 100644
--- a/src/worldfactory.cpp
+++ b/src/worldfactory.cpp
@@ -714,7 +714,7 @@ void worldfactory::load_last_world_info()
         translate_marker( "lastworld.json or one of its values is empty.  This could be due to data corruption or an unknown error and may be an indicator that your previously played world is damaged." );
 
     try {
-        if( !file_exist( lastworld_path ) ) {
+        if( !file_exist( lastworld_path ) || true ) {
             return;
         }
 
-- 
2.37.3.windows.1

